src/data/store/Tree.js

```diff
@@ -38,7 +38,16 @@ Ext.define('GeoExt.data.store.Tree', {
          *
          * @cfg {String}
          */
-        textProperty: 'name'
+        textProperty: 'name',
+
+        /**
+         * Changes behaviour of the checkbox of an ol.layer.Group (folder).
+         * Possible values are 'classic' or 'ol3'.
+         * 'classic' forwards the checkstate to the children of the folder.
```

Comment:
Please add more prose like 'If a folder is checked, then …, likewise if a folder is unchecked… If a layer in the folder is checked …`